.github/workflows/Tests.yml

```diff
@@ -48,36 +50,33 @@ jobs:
     name: ${{ matrix.type }} | ${{ matrix.test }}
 
     env:
-      PDK_ROOT: $GITHUB_WORKSPACE/rules/klayout
-      PDK: ${{ matrix.pdk }}
+      IMAGE: ghcr.io/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/test
 
     steps:
 
       - uses: actions/checkout@v3
 
-      - uses: actions/setup-python@v3
-        with:
-          python-version: '3.9'
-          cache: 'pip'
-          cache-dependency-path: '**/requirements.test.txt'
+      - name: Build container image
+        run: |
+          docker build -t "$IMAGE" -f- rules/klayout <<EOF
+          FROM gcr.io/hdl-containers/klayout
 
-      - run: pip install -r ./rules/klayout/requirements.test.txt
+          RUN apt-get update -qq \
+           && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+              make \
+              procps \
+              python3-pip \
+           && apt-get autoclean && apt-get clean && apt-get -y autoremove \
+           && rm -rf /var/lib/apt/lists/*
 
-      - name: Installing Klayout 0.27.8
-        run: |
-          sudo apt update -qq -y
-          wget https://www.klayout.org/downloads/Ubuntu-20/klayout_0.27.8-1_amd64.deb
-          sudo apt install -qq -y \
-            qt5-default \
-            qttools5-dev \
-            libqt5xmlpatterns5-dev \
-            qtmultimedia5-dev \
-            libqt5multimediawidgets5 \
-            libqt5svg5-dev
-          sudo dpkg -i ./klayout_0.27.8-1_amd64.deb
-          sudo apt-get install -f -y
+          RUN --mount=type=bind,target=/tmp/ctx pip3 install -r /tmp/ctx/requirements.test.txt
```

Comment:
why don't we install the requirements in the image as well?

Reply:
This line is precisely installing the requirements in the image. Note L61. L62-L72 is the content of the dockerfile provided through stdin.

Reply:
ah my bad! I missing the `RUN` before and got confused by the `--mount=type=bind`, it's an handy way to be able to use the context without `COPY`!